core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -171,6 +176,28 @@ public void killAll(Process proc, Map<String, String> modelEnvVars) throws Inter
             }
         return killers;
     }
+    
+    final List<ProcessKillingVeto> getVetoers() throws InterruptedException {
```

Comment:
This method does not contain invalidation logic for the cache. If somebody installs a new plugin for vetoing, it won't work till the class instance gets recreated